app/src/main/java/com/example/android/hilt/navigator/AppNavigatorImpl.kt

```diff
@@ -20,11 +20,12 @@ import androidx.fragment.app.FragmentActivity
 import com.example.android.hilt.R
 import com.example.android.hilt.ui.ButtonsFragment
 import com.example.android.hilt.ui.LogsFragment
+import javax.inject.Inject
 
 /**
  * Navigator implementation.
  */
```

Comment:
I know that you're following the codelab, but a good exercise it's to look at all the comments and think if it's really necessary to keep it or if the code is already self-explained